### PR TITLE
Test case on forgot password function

### PR DESCRIPTION
**Pull request by ninanichik**
This is positive test case for forgot password function. After inputting valid data test should be passed

### CODE REVIEW

File: oct/pages/forgot_password_page.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        return "Forgot Your Password?" in self._browser.title
+
+    def fill_email_on_forgot_password_page(self, email: str) -> None:
+        self._users_email.type_email(email)
+
+    def press_continue_button(self) -> None:
+        self._browser.find_element_by_xpath("//*[@id='content']/form/div/div[2]/input").click()
+
+
+class ConfirmationMsgWasSent(Page):
```
Comment (extsoft) [inline oct/pages/forgot_password_page.py]: @ninanichik Please rename this page (a name has to be noun). And use full names.

File: oct/pages/forgot_password_page.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        self._users_email = PersonalDetails(browser)
+
+    def open(self) -> None:
+        self._browser.get("https://localhost/index.php?route=account/forgotten")
+        self._browser.implicitly_wait(10)
+
+    def loaded(self) -> bool:
+        return "Forgot Your Password?" in self._browser.title
+
+    def fill_email_on_forgot_password_page(self, email: str) -> None:
```
Comment (extsoft) [inline oct/pages/forgot_password_page.py]: @ninanichik please rename to `fill_email`.

File: oct/pages/forgot_password_page.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+from selenium.webdriver import Remote
```
Comment (extsoft) [inline oct/pages/forgot_password_page.py]: @ninanichik please rename this file to `forgot_password.py`.

File: oct/pages/forgot_password.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        self._browser.find_element_by_xpath("//*[@id='content']/form/div/div[2]/input").click()
+
+
+class ConfirmationMessage(Page):
+    def __init__(self, browser: Remote):
+        self._browser = browser
+        self._browser.implicitly_wait(10)
+
+    def open(self) -> None:
+        self._browser.get("http://dp-151.zzz.com.ua/index.php?route=account/login")
```
Comment (extsoft) [inline oct/pages/forgot_password.py]: @ninanichik I expect to see `localhost` in the URL.

File: oct/pages/forgot_password.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        self._browser.implicitly_wait(10)
+
+    def loaded(self) -> bool:
+        return self._browser.find_element_by_css_selector(
```
Comment (extsoft) [inline oct/pages/forgot_password.py]: @ninanichik An exception will be thrown if the element is not found. So, this method can return `True` or thrown an error. But I think if you can't find an element, you have to return `False`.

File: oct/pages/forgot_password.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        self._browser = browser
+        self._browser.implicitly_wait(10)
+
+    def open(self) -> None:
+        self._browser.get("https://localhost/index.php?route=account/account")
+        self._browser.implicitly_wait(10)
+
+    def loaded(self) -> bool:
+        try:
+            return self._browser.find_element_by_css_selector(
```
Comment (extsoft) [inline oct/pages/forgot_password.py]: @ninanichik you have to return `bool` instead of `WebElement`.

Comment (ninanichik) [reply oct/pages/forgot_password.py]: @extsoft It's done

**Pull request merged**